setup.py

```diff
@@ -219,29 +126,45 @@ def check_import(pkgname, pkgver):
 for binary data.
 
 """,
-      classifiers=filter(None, classifiers.split("\n")),
-      author='Francesc Alted',
-      author_email='francesc@blosc.io',
-      maintainer='Francesc Alted',
-      maintainer_email='francesc@blosc.io',
-      url='https://github.com/Blosc/bcolz',
-      license='http://www.opensource.org/licenses/bsd-license.php',
-      # It is better to upload manually to PyPI
-      #download_url = 'http://github.com/downloads/Blosc/bcolz/python-bcolz
-      # -%s.tar.gz' % (VERSION,),
-      platforms=['any'],
-      install_requires=['cython>=' + min_cython_version, 'numpy>='+min_numpy_version],
-      cmdclass={'build_ext': build_ext},
-      ext_modules=[
-          Extension("bcolz.carray_ext",
-                    include_dirs=inc_dirs,
-                    define_macros=def_macros,
-                    sources=sources,
-                    library_dirs=lib_dirs,
-                    libraries=libs,
-                    extra_link_args=LFLAGS,
-                    extra_compile_args=CFLAGS)
-      ],
-      packages=['bcolz', 'bcolz.tests'],
-      package_data={'bcolz': ['carray_ext.pxd']},
+    classifiers=filter(None, classifiers.split("\n")),
+    author='Francesc Alted',
+    author_email='francesc@blosc.io',
+    maintainer='Francesc Alted',
+    maintainer_email='francesc@blosc.io',
+    url='https://github.com/Blosc/bcolz',
+    license='http://www.opensource.org/licenses/bsd-license.php',
+    # It is better to upload manually to PyPI
+    #download_url = 'http://github.com/downloads/Blosc/bcolz/python-bcolz
+    # -%s.tar.gz' % (VERSION,),
+    platforms=['any'],
+    ext_modules=[
+        Extension(
+            'bcolz.carray_ext',
+            include_dirs=inc_dirs,
+            define_macros=def_macros,
+            sources=sources,
+            library_dirs=lib_dirs,
+            libraries=libs,
+            extra_link_args=LFLAGS,
+            extra_compile_args=CFLAGS
+        )
+    ],
+    cmdclass={'build_ext': BuildExtNumpyInc},
+    install_requires=['numpy>=1.7'],
+    setup_requires=[
+        'cython>=0.22', 
+        'numpy>=1.7', 
+        'setuptools-scm>1.5.4'
+    ],
+    tests_require=tests_require,
+    extras_require=dict(
```

Comment:
Now that we have extras could you update the installation instructions to say something about:

```
pip install bcolz[extras]
```

And maybe also mention that bcolz releases can be installed via conda through the standard continuum channels.
(It usually takes a while for the to be built, experience shows, but usually not longer than a day)
```
conda install bcolz
```